src/oauth2/formats/html.ts

```diff
@@ -1,13 +1,14 @@
 import { render } from '../../templates';
 type KeyValue = { [key: string]: string };
 
-export function loginForm(msg: string, error: string, hiddenFields: KeyValue) {
+export function loginForm(msg: string, error: string, hiddenFields: KeyValue, registrationEnabled: boolean) {
 
   return render('login', {
     title: 'Login',
     msg: msg,
     error: error,
     hiddenFields: hiddenFields,
+    registrationEnabled,
```

Comment:
nitpick: why is this argument the only one not named?

Reply:
the others can be cleaned up as well

Reply:
I believe this is a new thing in ES6 where if key and value are the same, you can just directly call it without naming them?